app/src/main/java/com/example/cooky/ui/general/GeneralFragment.kt

```diff
@@ -96,6 +89,17 @@ class GeneralFragment private constructor(): BaseFragment<FragmentGeneralBinding
             }
 
             override fun onQueryTextChange(newText: String?): Boolean {
+                newText?.let {
+                    if(it.isNotEmpty()) {
+                        if(it[it.length - 1] == CHAR_SPACE) {
+                            viewModel.getAutoCompleteQuerys(it)
+                        }
+                    }
+                    else {
+                        bottomSheetBehavior.setVisible(false)
```

Comment:
viết như này có ổn k nhỉ? 
```suggestion
                        bottomSheetBehavior.visibility = false
```